src/confcom/azext_confcom/config.py

```diff
@@ -132,5 +132,9 @@
 SIDECAR_REGO_FILE = "./data/sidecar_rego_policy.txt"
 SIDECAR_REGO_FILE_PATH = f"{script_directory}/{SIDECAR_REGO_FILE}"
 SIDECAR_REGO_POLICY = os_util.load_str_from_file(SIDECAR_REGO_FILE_PATH)
+# api svn file
+SVN_API_FILE = "./data/svn_api"
```

Comment:
Not quite sure I understand all the context for this file, but this seems confusingly named. I believe it should be `version_api`?